sim/paladin/runes.go

```diff
@@ -84,6 +85,43 @@ func (paladin *Paladin) registerSheathOfLight() {
 	})
 }
 
+func (paladin *Paladin) registerShockAndAwe() {
+
+	if !paladin.hasRune(proto.PaladinRune_RuneCloakShockAndAwe) {
+		return
+	}
+
+	dep := paladin.NewDynamicStatDependency(
+		stats.Intellect, stats.SpellPower, 1.0,
```

Comment:
Based on the wowhead tooltip it seems like this should be `SpellDamage`. Their tooltips have been a bit of a mess lately though. Does it increase damage or damage+healing?